graph/src/data/subgraph/api_version.rs

```diff
@@ -50,7 +53,7 @@ impl UnifiedMappingApiVersion {
     ) -> Result<Self, DifferentMappingApiVersions> {
         let unique_versions: BTreeSet<Version> = versions.collect();
 
-        let all_below_referential_version = unique_versions.iter().all(|v| *v < API_VERSION_0_0_5);
+        let all_below_referential_version = unique_versions.iter().all(|v| *v < API_VERSION_0_0_6);
```

Comment:
These checks should not have been changed from 0.0.5, that's why there are test failures.